cms/static/js/views/pages/container.js

```diff
@@ -162,6 +173,15 @@ define(["jquery", "underscore", "gettext", "js/views/pages/base_page", "js/views
                 }
             },
 
+            updatePreviewButton: function(show_previews){
+                var text = (show_previews) ? gettext('Hide Previews') : gettext('Show Previews'),
+                    icon_class = (show_previews) ? 'icon-arrow-up' : 'icon-arrow-down',
+                    button = $('.nav-actions .button-toggle-preview');
+
+                this.$(".preview-arrow", button).removeClass("icon-arrow-down icon-arrow-up").addClass(icon_class);
```

Comment:
This JS and the button HTML needs to be updated to the new font-awesome classes. Right now the icon is not showing up.